### PR TITLE
BZ#1882203 - adding docs about openshift-master/redeploying-named-cer…

### PR DESCRIPTION
**Pull request by vwalek**
Signed-off-by: Vladislav Walek <22072258+vwalek@users.noreply.github.com>

BZ#1882203 - adding section about redeploying only named certificates.

@katherinedube please comment on changes

### CODE REVIEW

File: install_config/redeploying_certificates.adoc [hunk 1]
```diff
@@ -437,6 +437,25 @@ by deleting existing secrets that contain service serving certificates or removi
 annotations to appropriate services.
 ====
 
+[[redeploying-named-certificates]]
+=== Redeploying Named Certificates only
```
Comment (kalexand-rh) [inline install_config/redeploying_certificates.adoc:441]:
```suggestion
=== Redeploying Only Named Certificates
```

File: install_config/redeploying_certificates.adoc [hunk 2]
```diff
@@ -437,6 +437,25 @@ by deleting existing secrets that contain service serving certificates or removi
...
+
+The *_openshift-master/redeploy-named-certificates.yml_* playbook redeploys only named certificates. This also includes serial restarts of master services.
+
+To redeploy named certificates only, change the directory and run this playbook.
```
Comment (kalexand-rh) [inline install_config/redeploying_certificates.adoc:445]:
```suggestion
To redeploy named certificates only, change to the directory that contains the playbooks, and run this playbook.
```

**Review by kalexand-rh — COMMENTED** [review]
I have some text edits.

**Comment by kalexand-rh** [comment]
@gpei, will you PTAL?

**Comment by vwalek** [comment]
@kalexand-rh do you want me to squash the commits?

**Comment by kalexand-rh** [comment]
@vwalek, please do. (I'll still need to wait for QE to +1 the change before I merge.)

**Comment by vwalek** [comment]
@kalexand-rh done ;)

File: install_config/redeploying_certificates.adoc [hunk 3]
```diff
@@ -437,6 +437,25 @@ by deleting existing secrets that contain service serving certificates or removi
...
+
+----
+$ cd /usr/share/ansible/openshift-ansible
+$ ansible-playbook -i <inventory_file> \
+    playbooks/openshift-master/redeploy-named-certificates.yml
+----
+
+[NOTE]
+====
+The *_openshift_named_certificates_* inventory file must contain certificates with the same name as in the *_master-config.yaml_* file. If the names of `certfile` and `keyfile` are changed, you must update the xref:../install_config/certificate_customization.adoc#configuring-custom-certificates[named certificate parameters] in the *_master-config.yaml_* file on each master node and restart the `api` and `controllers` services. The `cafile` with the full ca chain is added to `/etc/origin/master/ca-bundle.crt`.
```
Comment (gpei) [inline install_config/redeploying_certificates.adoc:455]: Not quite sure what does this mean:
`The *_openshift_named_certificates_* inventory file must contain certificates with the same name as in the *_master-config.yaml_* file`
Maybe
`The *_ openshift_master_named_certificates_* parameter in ansible inventory file must contain certificates with the same name as in the *_master-config.yaml_* file` ?

Comment (vwalek) [reply install_config/redeploying_certificates.adoc:455]: @gpei sure, you want me to change it?

Comment (kalexand-rh) [reply install_config/redeploying_certificates.adoc:455]: @vwalek, please do. 👍

Comment (vwalek) [reply install_config/redeploying_certificates.adoc:455]: on it

**Comment by vwalek** [comment]
@katherinedube @gpei PTAL

**Comment by gpei** [comment]
LGTM

**Comment by katherinedube** [comment]
lgtm

**Pull request merged**